licenses/third-party.ftl

```diff
@@ -0,0 +1,48 @@
+<#-- To render the third-party file.
+ Available context :
+ - dependencyMap a collection of Map.Entry with
+   key are dependencies (as a MavenProject) (from the maven project)
+   values are licenses of each dependency (array of string)
+ - licenseMap a collection of Map.Entry with
+   key are licenses of each dependency (array of string)
+   values are all dependencies using this license
+-->
+<#function getUrl license>
+  <#if license == "Apache-2.0">
+    <#return "https://opensource.org/licenses/Apache-2.0">
+  <#elseif license == "MIT">
+    <#return "https://opensource.org/licenses/MIT">
+  <#elseif license == "BSD-3-Clause">
+    <#return "https://opensource.org/licenses/BSD-3-Clause">
+  <#elseif license == "EPL-2.0">
+    <#return "https://www.eclipse.org/legal/epl-2.0/">
+  <#elseif license == "GPLv2 with Classpath Exception">
+    <#return "https://www.gnu.org/software/classpath/license.html">
+  <#elseif license == "CDDLv1.0">
+    <#return "https://opensource.org/licenses/CDDL-1.0">
+  <#elseif license == "CDDLv1.1">
+    <#return "https://spdx.org/licenses/CDDL-1.1.html">
+  <#elseif license == "Bouncy Castle">
+    <#return "https://www.bouncycastle.org/licence.html">
+  <#else>
+    <#return "no known URL">
+  </#if>
+</#function>
```

Comment:
There are some more licenses we already support, let's add their URLs here. At some point in the future, we should maybe even move this out of here completely and consume it from the SDK.

```suggestion
<#function getUrl license>
  <#if license == "Apache-2.0">
    <#return "https://opensource.org/licenses/Apache-2.0">
  <#elseif license == "MIT">
    <#return "https://opensource.org/licenses/MIT">
  <#elseif license == "BSD-3-Clause">
    <#return "https://opensource.org/licenses/BSD-3-Clause">
  <#elseif license == "EPL-2.0">
    <#return "https://www.eclipse.org/legal/epl-2.0/">
  <#elseif license == "EPL-1.0">
    <#return "https://www.eclipse.org/legal/epl-v10.html">
  <#elseif license == "GPLv2 with Classpath Exception">
    <#return "https://www.gnu.org/software/classpath/license.html">
  <#elseif license == "GNU Lesser General Public License, Version 2.1">
    <#return "http://www.gnu.org/licenses/lgpl-2.1.html">
  <#elseif license == "CDDLv1.0">
    <#return "https://opensource.org/licenses/CDDL-1.0">
  <#elseif license == "CDDLv1.1">
    <#return "https://spdx.org/licenses/CDDL-1.1.html">
  <#elseif license == "Bouncy Castle">
    <#return "https://www.bouncycastle.org/licence.html">
  <#elseif license == "Go License">
    <#return "https://go.dev/LICENSE">
  <#else>
    <#return "no known URL">
  </#if>
</#function>
```